lib/assert.js

```diff
@@ -170,7 +170,12 @@ function _deepEqual(actual, expected, strict) {
              (expected === null || typeof expected !== 'object')) {
     return strict ? actual === expected : actual == expected;
 
-  // 7.5 For all other Object pairs, including Array objects, equivalence is
+  // 7.5. If both values are instances of typed arrays, wrap them in
```

Comment:
These numbers (`.7.5`) are not arbitrary! They come from [the CommonJS standard](http://wiki.commonjs.org/wiki/Unit_Testing/1.0#Assert). Looks like they may already be goobered up, but that's not a reason to make them further-removed from the standard they are intended to refer to.